### ISSUE TITLE
[HOLD for payment 2023-10-23] [HOLD for payment 2023-10-23] [$500] Chat - Entering @ in second line not showing 5 user suggestions

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#26480]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Tap on any report
3. Enter @ in compose to note 5 user suggestions displayed
4. Select any one
5. Tap enter key
6. In second line, enter @

## Expected Result:
Entering @ in second line must show 5 user suggestions

## Actual Result:
Entering @ in second line not showing 5 user suggestions

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [x] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.61-1

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/747b1db4-bb47-4d79-bc21-3a75be140e6e

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0107efd9d088b99c24</li>
        <li>Upwork Job ID: 1707566811859394560</li>
        <li>Last Price Increase: 2023-09-29</li>
<li>Automatic offers: </li>
<ul>
<li>cubuspl42 | Reviewer | 26993154</li>
</ul></li>
    </ul>
</details>

**Comment by alexpensify** [comment]
I'm OOO today and will test soon.

**Comment by alexpensify** [comment]
Closing - this is a dupe of https://github.com/Expensify/App/issues/26197

@allroundexperts - please share your proposal there for review. Thanks!

**Comment by cubuspl42** [comment]
@allroundexperts It looks like a dupe for me. Slightly different symptom of the same problem

**Comment by alexpensify** [comment]
Weekly Update: Putting this one on hold until https://github.com/Expensify/App/issues/26197 hits production and we can confirm this is still an issue.

**Comment by alexpensify** [comment]
Looks like I didn't edit the labels, this one should be a weekly.

**Comment by alexpensify** [comment]
@allroundexperts - this one is in staging, do you want to review if a PR is needed here and we can make this external again? Thanks!

**Comment by alexpensify** [comment]
@allroundexperts could you help to confirm if we should move this one off HOLD since the other GH is in staging? Thanks!

**Comment by alexpensify** [comment]
Thanks! Taking this off of HOLD, @cubuspl42 can you please review the proposal and see if it will resolve this issue?

**Comment by cubuspl42** [comment]
@allroundexperts You were right after all! Your solution looks good to me.

We should go with @allroundexperts

C+ reviewed 🎀 👀 🎀

**Comment by alexpensify** [comment]
@marcochavezf - we need a review here and we can make this one external again. Thanks for reviewing!

**Comment by marcochavezf** [comment]
Thanks for the review @cubuspl42, assigning @allroundexperts :rocket:

**Comment by alexpensify** [comment]
@allroundexperts any update on the PR for this one?

**Comment by alexpensify** [comment]
Weekly Update: It looks like the PR is moving forward!

**Comment by alexpensify** [comment]
@cubuspl42 - to prepare for the payment date, can you please complete the checklist? Thanks!

**Comment by cubuspl42** [comment]
@allroundexperts I modified the QA Steps from your PR to be more explicit, you can give it a quick look

**Comment by JmillsExpensify** [comment]
Can I get a payment summary on this issue?

**Comment by alexpensify** [comment]
Here is the payment summary: 

-  External issue reporter - N/A 
-  Contributor that fixed the issue - @allroundexperts $500
-  Contributor+ that helped on the issue and/or PR - @cubuspl42 $500 

Upwork Job: 

*If applicable, the bonuses will be applied on the final payment 

Extra Notes regarding payment: This solution took more than 6 business days, but I think the payment should include no penalty. The issue was fixed and took into account an edge case. I believe that we fixed this issue right instead of rushing, so keeping the payment as is for this one.
